scripts/utils-openssl.sh

```diff
@@ -73,7 +74,7 @@ install_openssl() {
         printf "Done.\n"
 
         printf "\tInstalling OpenSSL ${OPENSSL_TAG} ... "
-        make -j$NUMCPU install &>> $LOG_FILE
+        make -j$NUMCPU install 2>&1 >> $LOG_FILE
```

Comment:
```
In scripts/utils-openssl.sh line 77:
        make -j$NUMCPU install 2>&1 >> $LOG_FILE
               ^-----^ SC2086 (info): Double quote to prevent globbing and word splitting.
                               ^--^ SC2069 (warning): To redirect stdout+stderr, 2>&1 must be last (or use '{ cmd > file; } 2>&1' to clarify).
                                       ^-------^ SC2086 (info): Double quote to prevent globbing and word splitting.
```